exercises/gigasecond/canonical-data.json

```diff
@@ -1,45 +1,58 @@
 {
-   "#": [
-      "The basic test is to add one gigasecond to a few ordinary dates.",
-      "Most test programs currently check only the date and ignore the time.",
-      "For languages with a native date-time type though, expected times",
-      "here show the correct seconds, ignoring leap seconds.",
-      "The date and time formats here are per",
-      "http://www.ecma-international.org/ecma-262/5.1/#sec-15.9.1.15.",
-      "",
-      "Finally, as a demonstration but not really a test,",
-      "some languages demonstrate the add function by inviting the",
-      "solver to include their birthdate in either the solution code",
-      "or test program.  The test program then shows or tests their",
-      "gigasecond anniversay."
-   ],
-   "add": {
-      "description": [
-         "Add one gigasecond to the input."
-      ],
-      "cases": [
-         {
+  "exercise": "gigasecond",
+  "version": "1.0.0",
+  "comments": [
+    "The basic test is to add one gigasecond to a few ordinary dates.",
+    "Most test programs currently check only the date and ignore the time.",
+    "For languages with a native date-time type though, expected times",
+    "here show the correct seconds, ignoring leap seconds.",
+    "The date and time formats here are per",
+    "http://www.ecma-international.org/ecma-262/5.1/#sec-15.9.1.15.",
+    "",
+    "Finally, as a demonstration but not really a test,",
+    "some languages demonstrate the add function by inviting the",
+    "solver to include their birthdate in either the solution code",
+    "or test program.  The test program then shows or tests their",
+    "gigasecond anniversary."
+  ],
+  "cases": [
+    {
+      "add":
+      {
```

Comment:
I verified what happened locally, I did stashed, changed my local `$EXERCISE` value to review a different exercise, and verified it, merged it, grabbed the changes, went back to my branch, rebased it on master, unstashed my changes, forgot to reassign my exercise environment variable and then had it pass... of course...

So process error.